### PR TITLE
Add --direct to use geopmaccess without the service

### LINKED ISSUE #2777: As a user of `geopmaccess` I would like an option `--direct` that would read and write directly to `/etc/geopm-service`
**Is your feature request related to a problem? Please describe.**
I'm always frustrated that a functioning and active GEOPM Systemd Service is required in order to read and write the allowed list files.

**Describe the solution you'd like**
Provide a command line `--direct` / `-x` that will use the system_files.AccessLists object directly, rather than going through the DBus interface.  

**Describe alternatives you've considered**
Otherwise administrators will just write these files with a bash script, but there are permissions issues that may result from this solution.

**Additional context**
Partial implementation is here #2771

### PR DESCRIPTION
**Pull request by dannosliwcd**
- Resolves #2777
- When --direct is used, directly interact with GEOPM access lists instead of going through a service proxy.

### CODE REVIEW

File: service/geopmdpy/access.py [hunk 1]
```diff
@@ -392,10 +422,19 @@ def main():
...
+    if args.direct and not (args.group or args.default):
+        # This option is intended for early admin access list management before
+        # the GEOPM service is running. The option does not have a meaningful
+        # use case when run for the root user.
+        parser.error('Must specify either --group or --default with --direct.')
+
     try:
-        acc = Access(SystemMessageBus().get_proxy('io.github.geopm',
-                                                  '/io/github/geopm'))
+        geopm_proxy = (None if args.direct
```
Comment (dannosliwcd) [inline service/geopmdpy/access.py]: As an alternative to `None`, we could add a `DirectAccessProxy` that implements the proxy interface but makes direct calls. Doing so would mean we can avoid changes to the `TestAccess.py` file

Comment (dannosliwcd) [reply service/geopmdpy/access.py]: That would also remove all of the other `if none` checks throughout this file

Comment (cmcantalupo) [reply service/geopmdpy/access.py]: that seems like a cleaner approach.  I think we can wait until after monday for this change.  Thanks!

**Review by cmcantalupo — APPROVED** [review]
LGTM!

**Pull request merged**